packages/cardano-graphql-services/src/TxSubmit/cli.ts

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/* eslint-disable import/imports-first */
+require('../../scripts/patchRequire');
+import { Command } from 'commander';
+import { InvalidLoggerLevel } from '../errors';
+import { LogLevel, createLogger } from 'bunyan';
+import { TxSubmitHttpServer } from './TxSubmitHttpServer';
+import { URL } from 'url';
+import { loggerMethodNames } from '../util';
+import { ogmiosTxSubmitProvider } from '@cardano-sdk/ogmios';
+import onDeath from 'death';
+const clear = require('clear');
+const packageJson = require('../../package.json');
+clear();
+// eslint-disable-next-line no-console
+console.log('Tx Submit CLI');
+
+const program = new Command('tx-submit');
+
+program.description('Submit transactions to the Cardano network').version(packageJson.version);
+
+program
+  .command('start-server')
+  .description('Start the HTTP server')
+  .option('--url <url>', 'Server URL', (url) => new URL(url))
```

Comment:
Option 1 - single `url` parameter we parse

Reply:
I think if hostname won't be used for anything else, then a single `url` parameter is better. Otherwise individual parameters